test/Jamfile.v2

```diff
@@ -72,10 +72,9 @@ rule test_all
 
     if ! [ os.environ BOOST_LOG_TEST_WITHOUT_SELF_CONTAINED_HEADER_TESTS ]
     {
-        local headers_path = [ path.make $(BOOST_ROOT)/libs/log/include/boost/log ] ;
-        for file in [ path.glob-tree $(headers_path) : *.hpp : detail ]
+        for file in [ glob-tree-ex ../include/boost/log : *.hpp : detail ]
```

Comment:
What is `glob-tree-ex` and how is it different from `path.glob-tree`? Is this change necessary?

Also, I would prefer to keep the `headers_path` variable to avoid path duplication.